src/main/java/com/cmsc495/ticketsystem/controller/SubmitTicketController.java

```diff
@@ -64,9 +64,19 @@ public ResponseEntity<String> submitTicket(@RequestParam String name,
         }
     }
 
+
     private void sendEmail(TicketModel ticketModel, String email, String department, String issueType) {
         try {
-            emailService.sendEmail(email, department + issueType + ticketModel.getFormattedDate(), "Ticket submitted!");
+            // Construct a descriptive subject line
```

Comment:
This looks good to me, when you update from this review please attach a screenshot of a test email to display what the email looks like